src/org/parosproxy/paros/extension/encoder/Encoder.java

```diff
@@ -38,6 +38,7 @@
 
 import org.apache.log4j.Logger;
 
+import java.lang.*;

```

Comment:
Please use specific imports, not wildcards.

Reply:
- Actually this can be removed.
 - Should be added the ZAP comment to the class. [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines